.buildkite/hooks/environment

```diff
@@ -0,0 +1 @@
+export BUILDKITE_REPO=""
```

Comment:
Is it worth adding a comment here to indicate _why_ this is being set to an empty string?